packages/client/components/ActivityLibrary/ActivityDetailsSidebar.tsx

```diff
@@ -127,9 +127,7 @@ const ActivityDetailsSidebar = (props: Props) => {
   )
   const {onError, onCompleted, submitting, submitMutation, error} = useMutationProps()
   const history = useHistory()
-  const {togglePortal: toggleModal, modalPortal} = useModal({
-    id: 'createGcalEventModal'
-  })
+  const {isOpen: isScheduleModalOpen, open, close} = useDialogState()
```

Comment:
+1 I think `openDialog` and `closeDialog` would be more descriptive, otherwise I'm not sure what `open()` is opening on line 181